wormhole/module.py

```diff
@@ -422,6 +450,118 @@ async def remove_wormhole_slowmode(self, itx: commands.Context):
         storage.set(self, 0, key="wormhole_slowmode", value=0)
         await self._set_slowmode(0, itx)
 
+    @check.acl2(check.ACLevel.BOT_OWNER)
+    @wormhole_ban.command(
+        name="set",
```

Comment:
What about /wormhole ban, /wormhole unban and /wormhole banlist (with optional attributes, such as member for list of member bans, active for active bans and if there's no attribute, lists all)?